me.proton.Authenticator.yml

```diff
@@ -0,0 +1,49 @@
+app-id: me.proton.Authenticator
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '49'
+command: proton-authenticator
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.secrets
+  - --env=WEBKIT_DISABLE_DMABUF_RENDERER=1
+  # https://v2.tauri.app/plugin/single-instance/#flatpak
+  - --talk-name=me.proton.Authenticator.SingleInstance
+  - --own-name=me.proton.Authenticator.SingleInstance
+modules:
+  - name: proton-authenticator
+    buildsystem: simple
+    build-commands:
+      - bsdtar -Oxf ProtonAuthenticator.deb data.tar.gz | bsdtar -xf -
+      - install -Dm 755 "usr/bin/proton-authenticator" ${FLATPAK_DEST}/bin/proton-authenticator
+      - install -Dm 644 me.proton.Authenticator.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 "usr/share/applications/Proton Authenticator.desktop" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - desktop-file-edit --set-key=Exec --set-value='proton-authenticator %U' --set-icon=${FLATPAK_ID} --remove-key=Comment
+        "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 me.proton.Authenticator.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: file
+        dest-filename: ProtonAuthenticator.deb
+        only-arches: [x86_64]
+        url: https://proton.me/download/authenticator/linux/ProtonAuthenticator_1.1.4_amd64.deb
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Reply:
you can try generating a v1 lockfile and use that against the generator if you _really _ want to add this app here.

ideally someone would add support for v2+ lockfiles to the generator.